### PR TITLE
3.0.0-CI00044

### PR DESCRIPTION
**Pull request by erikzhang**

### CODE REVIEW

**Comment by vncoelho** [comment]
A historical achievement. Congratulations, Erik.

File: neo-cli/Shell/MainService.cs [hunk 1]
```diff
@@ -142,44 +145,27 @@ private bool OnBroadcastCommand(string[] args)
...
-            var tx = LoadScriptTransaction(
+            byte[] script = LoadDeploymentScript(
                 /* filePath */ args[1],
-                /* paramTypes */ args[2],
-                /* returnType */ args[3],
-                /* hasStorage */ args[4].ToBool(),
-                /* hasDynamicInvoke */ args[5].ToBool(),
-                /* isPayable */ args[6].ToBool(),
-                /* contractName */ args[7],
-                /* contractVersion */ args[8],
```
Comment (shargon) [inline neo-cli/Shell/MainService.cs]: all of this metadata, will be inside the smart contract?

File: neo-cli/Shell/MainService.cs [hunk 2]
```diff
@@ -815,19 +730,8 @@ private bool OnListAddressCommand(string[] args)
         private bool OnListAssetCommand(string[] args)
         {
             if (NoWallet()) return true;
-            foreach (var item in Program.Wallet.GetCoins().Where(p => !p.State.HasFlag(CoinState.Spent)).GroupBy(p => p.Output.AssetId, (k, g) => new
-            {
```
Comment (shargon) [inline neo-cli/Shell/MainService.cs]: we will add this in the future?

File: neo-cli/Shell/MainService.cs (hunk 1 above)
Comment (erikzhang) [reply neo-cli/Shell/MainService.cs]: They will be in manifest.

File: neo-cli/Shell/MainService.cs (hunk 2 above)
Comment (erikzhang) [reply neo-cli/Shell/MainService.cs]: Did not find a good way to list all NEP-5 assets.

Comment (shargon) [reply neo-cli/Shell/MainService.cs]: Maybe we should iterate all contracts, and try to find it according to NEP10?

Comment (shargon) [reply neo-cli/Shell/MainService.cs]: Or maybe we should add a new command for wallets, add nep5 token, and only look if you append it before?

Comment (erikzhang) [reply neo-cli/Shell/MainService.cs]: I will build this function into `Wallet`.

**Comment by shargon** [comment]
When https://github.com/neo-project/neo/pull/753 will be integrated, i can approve it

**Comment by erikzhang** [comment]
@shargon It will be. We are going to test it before approving.

**Comment by erikzhang** [comment]
I will draft a preview release later (after the pricing model). I think we can merge this first. @shargon @vncoelho

File: neo-cli/neo-cli.csproj [hunk 3]
```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <Copyright>2016-2019 The Neo Project</Copyright>
     <AssemblyTitle>Neo.CLI</AssemblyTitle>
-    <Version>3.0.0-preview1</Version>
+    <Version>2.10.1</Version>
```
Comment (shargon) [inline neo-cli/neo-cli.csproj]: Is not 3?

Comment (erikzhang) [reply neo-cli/neo-cli.csproj]: After we are ready for release the preview, I will increase the version number.

**Comment by shargon** [comment]
Maybe we need to solve the test error before merge

**Comment by erikzhang** [comment]
The error is because of the wrong uploaded package on the NuGet. There is no way to solve it now.

**Pull request merged**